gluten-ut/spark33/src/test/scala/org/apache/spark/sql/execution/GlutenReplaceHashWithSortAggSuite.scala

```diff
@@ -85,7 +85,7 @@ class GlutenReplaceHashWithSortAggSuite
     withTempView("t1", "t2") {
       spark.range(100).selectExpr("id as key").createOrReplaceTempView("t1")
       spark.range(50).selectExpr("id as key").createOrReplaceTempView("t2")
-      Seq("COUNT", "COLLECT_LIST").foreach {
+      Seq("COLLECT_LIST").foreach {
```

Comment:
Since `ShuffledHashJoinExec` becomes `SortMergeJoinExec`, `COUNT` remains consistent with the vanilla spark suite. For more clarity, I added the `COUNT` suite.